util/linux_install.sh

```diff
@@ -90,11 +90,22 @@ elif grep ID /etc/os-release | grep -q gentoo; then
 			sys-devel/gcc \
 			sys-devel/crossdev dev-embedded/avrdude
 		sudo crossdev -s4 --stable --g =4.9.4 --portage --verbose --target avr
-		echo Done!
+		echo "Done!"
 	else
 		echo "Quitting..."
 	fi
 
+elif grep ID /etc/os-release | grep -q sabayon; then
+	sudo equo install \
+		app-arch/unzip \
+		app-arch/zip \
+		app-mobilephone/dfu-util \
+		net-misc/wget \
+		sys-devel/gcc \
+		sys-devel/crossdev dev-embedded/avrdude
```

Comment:
```suggestion
		sys-devel/crossdev
```
Just one more super minor nitpick. Split this line so that every package is on a separate line. Put `dev-embedded/avrdude \` above the `net-misc/wget \` line so that dependencies are sorted alphabetically.

Other than that, LGTM.

Reply:
And apply the same changes to the gentoo block too, I presume?

Reply:
Yep, that'd be great, thanks!

Reply:
@BlitzKraft You can use the "Apply suggestion" option in the UI, and that will immediately commit the change to the branch. 

This is more of an FYI, since it's a new feature, and AMAZING.

Reply:
Yeah. I just noticed.